browser/src/map/handler/Map.WOPI.js

```diff
@@ -449,7 +449,11 @@ L.Map.WOPI = L.Handler.extend({
 		}
 		else if (msg.MessageId === 'Action_ChangeUIMode') {
 			if (msg.Values && msg.Values.Mode !== null && msg.Values.Mode !== undefined) {
-				this._map.fire('changeuimode', {mode: msg.Values.Mode, force: false});
+				if (msg.Values.Mode === 'notebookbar') {
```

Comment:
shouldn't be `msg.Values.Mode === 'notebookbar' || msg.Values.Mode === 'tabbed'` ?